### DISCUSSION TITLE
[Question] GitHub Action .exe icon

### DISCUSSION THREAD

**Discussion by God-damnit-all** [legendary-gl/legendary#356] [Q&A] [answered]
I'm just curious, why does the legendary.exe file built by the workflow have a different icon than the one uploaded to Releases? Intentional?

**Comment by derrod** [comment, marked as answer]
Because the workflow doesn't have the Legendary icon files. I build release binaries locally since I have to do that for Linux and PyPI anyway and for that I include the icon.

**Comment by God-damnit-all** [comment]
I keep forgetting Discussions on GitHub is a thing, sorry about that.

Thank you.
